pkg/proto/pbgo/core/api.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.0
-// 	protoc        v5.26.1
+// 	protoc-gen-go v1.28.1
```

Comment:
It seems that the protoc-gen-go use to regenerate the proto *.go file is an older version than during the previous generation. 
Could you check if it is expected and how the version is managed?

Reply:
Based on this [README.md](https://github.com/DataDog/datadog-agent/blob/946ce87619db16edd9002965f9146afada978f98/pkg/proto/datadog/api/README.md?plain=1#L3), it seems like the version is intentionally pinned to 1.28.1

The commit history alternates quite a bit between 1.28.1 and 1.3* versions. Not entirely sure how `protoc-gen-go` versions past 1.28.1 are being used to generate the `.pb.go` files since the commands in `inv generate-protobuf` are deprecated in newer versions. Asking some of the authors of recent commits how/why they used newer versions of `protoc-gen-go`.